lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -1387,6 +1387,7 @@ unsafe fn from_go_array<T: Clone>(data: *mut T, len: u32) -> Vec<T> {
 pub enum CGOErrCode {
     ErrCodeSuccess = 0,
     ErrCodeFailure = 1,
+    ErrCodeClientPtr = 2,
```

Comment:
perhaps some comment indicating what this error means, when to return it, and what to do when something returns it to you would be appropriate?